aws-bundle/build.gradle

```diff
@@ -27,6 +27,7 @@ project(":iceberg-aws-bundle") {
     implementation platform(libs.awssdk.bom)
     implementation "software.amazon.awssdk:apache-client"
     implementation "software.amazon.awssdk:auth"
+    implementation "software.amazon.awssdk:http-auth-aws-crt"
```

Comment:
adding this requires also an update to the `LICENSE` file

Reply:
Sure. I'll update in the next commit